scripts/process_conv_pp.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Feb 12 10:40:09 2021
+
+@author: Julius
+"""
+
+import sys
+import pandas as pd
+
+in_path = sys.argv[1] + '\conventional_power_plants_DE.csv'
```

Comment:
In windows, a backslash is used to separate directories in paths, in linux a slash. To write code that runs independently from the os, use pythons `os.path.join()`.

Reply:
Apart from this, it is better to expect the full path being passed as a console argument, including the name of the file.